src/main/java/com/google/sps/data/TSP.java

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.*;
+
+/** Represents a comment in store or user page. */
+public class TSP {
+
+  // private Map<String, Integer> indexToStoreId = new HashMap<>();
+
+  public static double[][] constructAdjacencyMatrix(
+      List<Map<String, String>> coffeeShops, double userLat, double userLng) {
```

Comment:
To improve readability, suggest using a map name that includes the key and value e.g. <key>To<value>Map e.g storeIdToStoreMap.

Reply:
I'm not too sure what I should change it to since the Map is converted from a JSON string and each map contains name, distance to user, storeID, etc.